derp/derp_server.go

```diff
@@ -960,7 +967,7 @@ type dropReason int
 const (
 	dropReasonUnknownDest      dropReason = iota // unknown destination pubkey
 	dropReasonUnknownDestOnFwd                   // unknown destination pubkey on a derp-forwarded packet
-	dropReasonGone                               // destination tailscaled disconnected before we could send
+	dropReasonGoneDisconnected                   // destination tailscaled disconnected before we could send
```

Comment:
There is no dropReasonGoneUnknown here because we don't actually have any such cases in the current code?

(Note: if this comment seems nonsensical, it probably is. I don't understand this part of the code very well)

Reply:
I *think* that the already existing "dropReasonUnknownDest" and "dropReasonUnknownDestOnFwd" are the variables that were already being incremented for this case (the ones described in the original issue text) so all we need to do is actually send a peerGone packet back to the client.